app/scenes/CollectionDelete.tsx

```diff
@@ -53,6 +55,19 @@ function CollectionDelete({ collection, onSubmit }: Props) {
             }}
           />
         </HelpText>
+        {team.defaultCollectionId === collection.id ? (
+          <HelpText>
+            <Trans
+              defaults="Also, <em>{{collectionName}}</em> is being used as a Default Collection. Deleting it will reset the Default Collection to Home page."
```

Comment:
```suggestion
              defaults="Also, <em>{{collectionName}}</em> is being used as the start view – deleting it will reset the start view to the Home page."
```